styles/index.scss

```diff
@@ -625,6 +638,46 @@ button.UNX-page-next {
   font-weight: 600;
   line-height: 24px;
 }
+.UNX-facets-all {
+  transition: all .3s ease-in;
+}
+.UNX-accordian-close .UNX-facets-all {
+  max-height: 0px;
+  overflow: hidden;
+  opacity: 0;
+}
+.UNX-facets-item.UNX-view-more {
+  height: 100px;
+}
+.UNX-facets-item {
+  height: auto;
+  overflow:scroll;
+}
+.UNX-accordian-open .UNX-facets-all {
+  max-height: 300px;
+  opacity: 1;
+  overflow: scroll;
+}
+.view-more-row {
+  display: flex;
+  justify-content: flex-end;
+}
+.view-more-row button {
```

Comment:
can we give a class to the button & use that instead? this gives us the flexibility to change the HTML tag from button to anything else later without updating the CSS

Reply:
done